examples/aws/storage-efs/README.md

```diff
@@ -0,0 +1,242 @@
+```
+aws efs create-mount-target \
+    --file-system-id $file_system_id \
+    --security-groups $security_group_id \
+    --subnet-id subnet-EXAMPLEe2ba886490
+```
+
+
+## 6.0 Using the EFS Storage in Kubeflow
+
+### Sample 1 - Static Provisioning
+[Using this sample from official AWS Docs](https://github.com/kubernetes-sigs/aws-efs-csi-driver/tree/master/examples/kubernetes/multiple_pods) we have provided the required spec files in the sample subdirectory but you can create the PVC another way. 
+
+1. Use the `$file_system_id` you recorded before or use the following command to get the efs filesystem id - 
+```
+aws efs describe-file-systems --query "FileSystems[*].FileSystemId" --output text
+```
+
+2. Now edit the sample/pv.yaml and edit the `volumeHandle` to point to your EFS filesystem
+
+3. Now create the required persistentvolume, persistentvolumeclaim and storageclass resources as -
+```
+kubectl apply -f examples/aws/storage-efs/sample/pv.yaml
+kubectl apply -f examples/aws/storage-efs/sample/pvc.yaml
+kubectl apply -f examples/aws/storage-efs/sample/sc.yaml
+```
+
+
+## 7.0 Connecting to Central dashboard
+Port Forward as needed and Login to http://localhost:8080 using default credentials.
+
+## 8.0 Test your Setup
+Check the `Volumes` tab in Kubeflow and you should be able to see your PVC is available for use within Kubeflow as follows - 
+
+### 8.1 Additional Permissions 
+You might need to specify some additional directory permissions on your worker node before you can use these as mount points. The set-permission-job.yaml is an example of how you could set these permissions to be able to use the efs as your workspace in your kubeflow notebook. Edit the spec file as needed for your usecase
+```
+kubectl apply -f examples/aws/storage-efs/sample/set-permission-job.yaml
+```
+
+### 8.2 Using Kubeflow Notebooks
+1. Spin up a new Kubeflow notebook server and specify the name of the PVC to be used as the workspace volume or the data volume and specify your desired mount point. 
+2. In case the server does not start up in the expected time, do make sure to check - 
+    - The Notebook Controller Logs
+    - The specific notebook server instance pod's logs
+
+### 8.3 Training using TFJob Operator
+The following section re-uses the PVC and the Kubeflow Notebook created in the previous steps to download a dataset to the EFS Volume. Then we spin up a TFjob which runs a image classification job using the data from the shared volume. 
+Source: https://www.tensorflow.org/tutorials/load_data/images
+
+1. Download the dataset to the EFS Volume using the kubeflow notebook created above 
+```
+import pathlib
+import tensorflow as tf
+dataset_url = "https://storage.googleapis.com/download.tensorflow.org/example_images/flower_photos.tgz"
+data_dir = tf.keras.utils.get_file(origin=dataset_url,
+                                   fname='flower_photos',
+                                   untar=True)
+data_dir = pathlib.Path(data_dir)
```

Comment:
can you help me understand how is it pushed to volume?

Reply:
Since we are using the kubeflow notebook from the previous section which has the EFS volume mounted as personal workspace at `/home/jovyan`, this is the path we are in.
Thus when running the TF notebook from here, keras downloads the data directory into `/home/jovyan/.keras`

I definitely need to add more specific details to the README.